workspaces/redhat-resource-optimization/package.json

```diff
@@ -59,7 +60,8 @@
     "@types/react": "^18",
     "@types/react-dom": "^18",
     "react": "^18",
-    "react-dom": "^18"
+    "react-dom": "^18",
+    "@backstage/backend-plugin-api": "^1.1.0"
```

Comment:
Why is this here?

Reply:
@jkilzi after adding the rbac plugin as dependency I was facing issues while running the backend due to different versions of  `@backstage/backend-plugin-api` hence I added above.